proms-resources/clinical-data/src/main/resources/SLING-INF/content/Extensions/AdminDashboard/Clinics.json

```
@@ -4,6 +4,6 @@
   "cards:extensionName": "Clinics",
   "cards:targetURL": "/content.html/admin/Clinics",
   "cards:icon": "asset:proms-homepage.clinicIcon.js",
-  "cards:hint": "Configure clinics to be queried.",
-  "cards:defaultOrder": 30
-}
\ No newline at end of file
+  "cards:hint": "Configure the registered clinics",
+  "cards:defaultOrder": 410
```

I would put this right after `Groups`, since I view clinics not a PROMS-only thing, but an entity above a group.

---

Okay, this can be changed to 150 then.
```suggestion
  "cards:defaultOrder": 150
```